compliance/cis_k8s/rules/cis_1_1_20/rule.rego

```diff
@@ -0,0 +1,18 @@
+package compliance.cis_k8s.rules.cis_1_1_20
+
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the Kubernetes PKI certificate file permissions are set to 644 or more restrictive (Manual)
+finding = result {
```

Comment:
nit: please separate to three parts to make it easier to read: `# filter` - `# evaluate` - `# set result`